### ISSUE TITLE
Reopen Tray-ed instance when opening desktop file

### ISSUE THREAD

**Issue by jayghoshter** [intersimone999/pyqtws#12]
Hi,

Great software. 

Is there a way to reopen a trayed instance of an app by executing the .desktop file again? Currently, it opens a new instance, and the only way to reopen the old instance is by clicking the tray icon. 

This is super helpful for fully keyboard-based workflows. 

Thanks!

**Comment by intersimone999** [comment]
I agree: apps with the TrayIcon plugin should not open twice. This could be implemented via dbus, I guess.
